rskj-core/src/main/java/co/rsk/net/discovery/NodeChallengeManager.java

```diff
@@ -29,9 +31,15 @@
  * Created by mario on 22/02/17.
  */
 public class NodeChallengeManager {
+    private static final Logger logger = LoggerFactory.getLogger(NodeChallengeManager.class);
+
     private Map<String, NodeChallenge> activeChallenges = new ConcurrentHashMap<>();
 
     public NodeChallenge startChallenge(Node challengedNode, Node challenger, PeerExplorer explorer) {
+        logger.debug("co.rsk.net.discovery.NodeChallengeManager.startChallenge - Starting challenge for node:\n" +
+                "challengedNode: {}\n" +
+                "challenger: {}", challengedNode.toString(), challenger.toString());
```

Comment:
I think it's no need to call `toString()` on objects you want to log - it'll be done under the hood automatically. Plus `null`'s would be handled correctly, plus the `toString()` method wouldn't be called at all for higher log levels

Reply:
Done, I removed the `toString()` calls.